osh.spec

```diff
@@ -298,6 +309,11 @@ fi
 %postun hub
 %systemd_postun osh-{retention,stats}.{service,timer}
 
+%files worker-manager
+%defattr(-,root,apache,-)
```

Comment:
The use of `%defattr` seems undesired to me.  The binary is readable/executable by anyone, so it does not need to be in the `apache` group.  Same with the `/etc/osh/worker-manager` directory.  So I think that the `%defattr` line could/should be removed.